apps/cli/cli_common.c

```diff
@@ -774,6 +774,73 @@ cli_set_mode(clixon_handle h,
     return retval;
 }
 
+/*!
+ * @brief Runs a Python script in a new process
+ *
+ * @param[in] h Clicon handle
+ * @param[in] cvv CLIgen variable vector
+ * @param[in] argv function arguments
+ * @retval     0    OK
+ * @retval    -1    Error
+ * The function creates a new child process in which the Python script is launched.
+ * In the child process, before launching the script, environment variables are set
+ * taken from cvv.
+ *
+ * The function returns the exit code of the Python script, or a value of (-1) on error.
+ */
+int
+cli_start_python3(clixon_handle h,
```

Comment:
May I ask what is the reason for this feature?
It can be a security problem when you allow to execute an arbitrary Python script.
In any case, the decision of which runner to use is normally defined by the script's shebang. Maybe no need to define the runner name (`python3` in this case). Instead, just pass the script name to the exec() call. In this case, the function will allow running any executable (not just a Python script).

Reply:
That's the point, the administrator can specify a script in the cli that can be executed and if something goes wrong, then we will return to the cli, and if we do system(), then bash starts, and then the script, and if something goes wrong, then we get into bash, this is a security problem It's not in my solution. it is also assumed that the user is only in the cli and does not have access to the system, but maybe I am outdated and something has changed :)

Reply:
@khromenokroman 
In any case, why not rely on Script's shebang string?

Reply:
Sorry, I took into account all the comments and changed the code, but left the description the same, now I have made changes to the description of the function